### DISCUSSION TITLE
Token not deposited to related accout

### DISCUSSION THREAD

**Discussion by JohnFnc** [blockscout/blockscout#9400] [Support] [answered]
I transfered from safe account to Mexc account but it is not deposited.

https://safe.zetachain.com/transactions/tx?id=multisig_0xbC882a272D52C67dBD245a3ff5acDfD957626973_0x9897dbd21398712f9b44a1bbfacdab5fd31f6e7cfdf6332f5ecc3bdb03798208&safe=zetachain-mainnet:0xbC882a272D52C67dBD245a3ff5acDfD957626973

This Tx doesnt seem as a mexc account so I cant proof it.Could you please support.

Best Regards

**Comment by ekawibisono** [comment, marked as answer]
Hello, This is blockscout github, not zetachain, please you call on development [zetachain Community](https://www.zetachain.com/community)

Thanks
